docs/buildah-run.md

```diff
@@ -0,0 +1,37 @@
+## buildah-run "March 2017"
+
+## NAME
+buildah run - Run a command inside of the container. 
+
+
+## SYNOPSIS
+**buildah** **run** **containerID** [*command options* [...]] 
+
+## DESCRIPTION
+Runs a specified command using the container's root filesystem as a root filesystem, using configuration settings inherited from the container's image or as specified using previous calls to the config command
```

Comment:
So if I change:  "Runs a specified command using the container's..." to "Launches a container and runs the specified command in that container using the container's...".  Would that address this concern?  I'll make it so, holler if it needs adjusting.